### PR TITLE
Bug 1557218 Added section to upgrade docs about swap memory

### PR DESCRIPTION
**Pull request by bfallonf**
For: https://bugzilla.redhat.com/show_bug.cgi?id=1557218

cc @wmengRH @jiajliu 

Will ask for an ack in the BZ.

### CODE REVIEW

File: upgrading/index.adoc [hunk 1]
```diff
@@ -81,6 +81,19 @@ in-place upgrades or blue-green deployments.
...
 Ansible-based upgrade and provides the equivalent commands to run manually. Using this method describes the in-place xref:install-config-upgrading-strategies[upgrade strategy].
 
+[[upgrading-swap-memory]]
+== Considerations for upgrading with swap memory
+
+A clean installation of {product-title} version 3.9 has swap memory enabled,
+by having the `openshift_disable_swap` setting in the Ansible install file set
+to `false` by default.
+
+If you had enabled swap memory, then the upgrade to {product-title} version 3.9
```
Comment (jianlinliu) [inline upgrading/index.adoc]: How about replace the above lines like the following (no need to mention 3.9 fresh install):
A existing old version of cluster may have swap memory enabled (by having the `openshift_disable_swap` setting to false in the Ansible inventory file or manually enabled swap in the cluster). If that, the upgrade to .....

**Comment by bfallonf** [comment]
Thanks @jianlinliu . I've updated to your suggestion. If there's anything else, please let me know.

In the meantime @openshift/team-documentation PTAL

**Comment by jianlinliu** [comment]
LGTM.

File: upgrading/index.adoc [hunk 2]
```diff
@@ -81,6 +81,18 @@ in-place upgrades or blue-green deployments.
 The manual upgrade method breaks down the steps that happen during an automated
 Ansible-based upgrade and provides the equivalent commands to run manually. Using this method describes the in-place xref:install-config-upgrading-strategies[upgrade strategy].
 
+[[upgrading-swap-memory]]
+== Considerations for upgrading with swap memory
+
+If you are upgrading an existing, older version of an {product-title} cluster,
```
Comment (kalexand-rh) [inline upgrading/index.adoc]: I might say:
If you are upgrading an existing, older version of an {product-title} cluster,
you might have swap memory enabled. You can enable swap memory by setting the `openshift_disable_swap` parameter in the Ansible inventory file to `false` or manually enabling swap
in the cluster. If you enabled swap memory, then upgrading to {product-title} version 3.9 will
fail. To successfully upgrade to {product-title} version 3.9, ensure swap memory
is disabled by following the
xref:../../admin_guide/overcommit.adoc#disabling-swap-memory[Disabling Swap
Memory] section of the Cluster Administration guide.

**Comment by bfallonf** [comment]
Thanks @kalexand-rh . I didn't want to put in a "You can" statement, because it might cause some "how do I do it?" questions, but I took in your "If you enabled swap memory..." change. That's a better suggestion.

I'll merge.

**Comment by kalexand-rh** [comment]
That makes sense, Brice. Thanks!

**Pull request merged**